### PR TITLE
HPCC-24900 Add Container Cost Section to Docs

### PR DESCRIPTION
**Pull request by g-pan**
Signed-off-by: g-pan <greg.panagiotatos@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [x] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:

Successful integration testing:  http://10.224.20.18/view/Docs-gp/job/DocBuild-01-GPRepo/20/

### CODE REVIEW

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
All in all, this is very good. A few comments are inline regarding presentation and formatting.

**Review by shamser — APPROVED** [review]
Agree with James' suggestion. Otherwise looks goods.

**Review by shamser — APPROVED** [review]

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml [hunk 1]
```diff
@@ -496,4 +496,372 @@ eclagent:
...
+
+      <para>The new cost field is shown in the Logical File summary page. It
+      is the combined cost of storing and accessing the data.</para>
+
+      <para>The cost information is currently only generated for Thor and
+      hThor jobs.</para>
+    </sect2>
+  </sect1>
+
+  <sect1 id="CostsConfigurations">
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml:655]: This should be a sect2, as we discussed offline

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
One organizational thing, see comment inline

File: docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml [hunk 2]
```diff
@@ -496,4 +496,372 @@ eclagent:
...
+
+        <para><informaltable colsep="1" frame="all" rowsep="1">
+            <tgroup cols="2">
+              <colspec align="left" colwidth="122.40pt" />
+
+              <colspec />
+
+              <tbody>
+                <row>
+                  <entry><emphasis>limit </emphasis></entry>
```
Comment (JamesDeFabia) [inline docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml:799]: Why are these numbered?

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
another comment inline

**Review by JamesDeFabia — APPROVED** [review]
Looks good

**Pull request merged**